packages/manager/modules/nasha/src/order/order.routing.js

```diff
@@ -0,0 +1,18 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('nasha.order', {
+    url: '/order',
+    component: 'nashaOrder',
+    resolve: {
+      breadcrumb: () => null,
+      cancelLink: /* @ngInject */ ($state) => $state.href('nasha'),
+      plans: /* @ngInject */ ($http, coreConfig, preparePlans) =>
+        $http
+          .get(
+            `/order/catalog/public/f4a53r9w?ovhSubsidiary=${
```

Comment:
is it the definitve product name ?

Reply:
`nasha` is and will be the product name in production
`f4a53r9w` is the temporary product name to test the new order funnel